app/Resources/views/full/bundle_list.html.twig

```diff
@@ -0,0 +1,81 @@
+{% extends 'pagelayout.html.twig' %}
+
+{% block content %}
+    <div class="bundles-list">
+        <div class="bundles-list-header">
+            <div class="container">
+                <div class="row">
+                    <div class="col-xs-12">
+                        <p class="bundles-list-hello-message">Search for the bundle you need or add a new one and contribute to our software</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="bundles-list-content">
+            <div class="container">
+                <div class="col-xs-12 bundles-list-container">
+                {% include 'parts/bundle_list/list.html.twig' with {
+                    'items': items,
+                    'viewType': 'line',
+                    'extraParams': {
+                        'page': 1
+                    }
+                } %}
+                </div>
+                <div class="bundles-list-load-more">
+                    {% if extraParams.totalCount > items|length %}
+                        <button onclick="loadBundleCards()">Load more</button>
+                    {% endif %}
+                </div>
+            </div>
+        </div>
+    </div>
+    <script>
+        {% autoescape 'js' %}
+        var page = 1,
```

Comment:
please wrap the code with 
```
$(document).ready(function() { });   
or
$(function() { });   
```
to be sure that DOM will be ready for jquery selectors